distribution/src/main/resources/config/jvm.options

```diff
@@ -44,9 +44,6 @@
 
 ## basic
 
-# force the server VM
--server
```

Comment:
I didn't see this the first time, but I notice that you removed it here and from the windows bat file, is there a reason we no longer force server mode?

Reply:
@dakrone The Windows service can not handle the `-server` JVM option, so we had to filter it out there. However, we have a client JVM bootstrap check and `-server` is the default on all JVMs that we support these days, so I do not think we need to force this flag anymore. This means that we do not need to filter this out in the JVM options parsing on the Windows service only, simplifying the JVM options parsing which was a side-benefit of this effort.